components/atoms/Pill/pill.tsx

```diff
@@ -8,9 +9,10 @@ export interface PillProps {
   color?: "slate" | "green" | "yellow" | "red" | "purple";
   size?: "base" | "small" | "xsmall";
   icon?: JSX.Element;
+  blurText?: boolean;
 }
 
-const Pill: React.FC<PillProps> = ({ className, text, color = "slate", size = "base", icon }) => {
+const Pill: React.FC<PillProps> = ({ className, text, color = "slate", size = "base", icon, blurText = false }) => {
```

Comment:
I added a `blurText` prop to the Pill component as I need to blur text, not the pill for the OSCR component.